JS_HW/task_6.js

```diff
@@ -0,0 +1,38 @@
+// Написать функцию которая принимает в себя строчное значение состоящее из одного слова. В слове могут быть буквы разных регистров. Результатом выполнения данной функции должна быть строка, в которой первая буквы заглавная, а все последующие в нижнем регистре.
+// Если в слове четное количество букв, то и последняя буква должна быть заглавной.
+// Пример:
+
+// someFn(‘пиТеР’) // Питер
+// someFn(‘javaScript’) // JavascripT
+
+
+
+
+
+
+
+
+function someFn (string) {
+    const toLowerCase = string.toLowerCase();
+    const firstBigLetter = toLowerCase[0].toUpperCase() + toLowerCase.slice(1);
```

Comment:
naming issue. The name of your variable is telling that it stores letter, but in your case it stores whole word